packages/mds-metrics/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { GetTimeBinsParams } from './types'
+
+export function getTimeBins(params: GetTimeBinsParams) {
+  const { start_time, end_time, bin_size } = params
```

Comment:
Could do this destructuring directly in the function signature if you prefer:
```
export function getTimeBins({start_time, end_time, bin_size}: GetTimeBinsParams) {...}
```
You could also use `Partial<GetTimeBinsParams>` and assign the defaults right inline as well.